code/game/turfs/closed/minerals.dm

```diff
@@ -80,6 +82,23 @@
 	else
 		return attack_hand(user)
 
+/turf/closed/mineral/attack_hand(mob/user)
+	if(weak_turf)
+		var/turf/T = user.loc
+		if (!isturf(T))
+			return
+		if(last_act + 20 > world.time)//prevents message spam
+			return
+		last_act = world.time
+		to_chat(user, "<span class='notice'>You start pulling out pieces of rock with your hands...</span>")
+
+		if(do_after(user, 60, target = src))
```

Comment:
This is too fast and should use the time defines. That goes for most of the code, seems to be a lack of time defines (using SECONDS define)

Reply:
Ah sorry most of this is old and using the old style of megafauna sleep type stuff if that makes sense.